### DISCUSSION TITLE
Question about how the "vocab" created.

### DISCUSSION THREAD

**Discussion by Alex2975** [bowang-lab/scGPT#107] [Q&A]
Dear Authors, 

Thank you very much for making this tool public. Very impressive. I have a question about the vocab.json. The gene names are used as tokens. How did you created the vocab.json? I search for some of the gene names in the json file, and I cannot find the exact names. Could you please share how the gene names were collected for training the base model? If I have a new scRNAseq data set, how can I map my gene Ensembl IDs to the gene names in the vocab.json? Thank you again.

**Comment by subercui** [comment]
Hi, since we collected the data from CellXGene, we followed their gene names there as well. You can find a mapping between Ensembl IDs to the names in the attached file

[gene_info.csv](https://github.com/bowang-lab/scGPT/files/13243634/gene_info.csv)

**Reply by Alex2975** [reply]
Thank you so much for the help. I will try the csv file.
